README.md

```diff
@@ -37,6 +36,34 @@ fastify.get('/login/facebook/callback', async function (request, reply) {
 
   reply.send({ access_token: result.access_token })
 })
+
+fastify.register(oauthPlugin, {
+	name: 'githubOAuth2',
```

Comment:
Could you use double spaced instead of a tab?
So the example could be more readable and "pastable copy"